v2/apiserver/internal/core/jobs_test.go

```diff
@@ -210,6 +210,52 @@ func TestJobsServiceCreate(t *testing.T) {
 				require.Contains(t, err.Error(), "error saving event")
 			},
 		},
+		{
+			name: "error storing job environment in substrate",
+			service: &jobsService{
+				eventsStore: &mockEventsStore{
+					GetFn: func(context.Context, string) (Event, error) {
+						return Event{
```

Comment:
nit: wondering if this extra worker spec config is relevant to the tests at hand?   If not, perhaps we remove in the name of brevity?

Reply:
At a glance, it doesn't look like it needs to be there. I am scratching my head as to why I ever had it there to begin with. I'll shorten this as suggested.

Reply:
Didn't hit me until I went to change it, but there _is_ a reason this was there.

Starting on line 363, this hardcoded `Job` is the used for _all_ the test cases:

```
Job{
	Spec: JobSpec{
		PrimaryContainer: JobContainerSpec{
			Privileged:          true,
			UseHostDockerSocket: true,
			WorkspaceMountPath:  "/var/workspace",
		},
	},
}
```

This job has all the bells and whistles so that various test cases have the opportunity to trip over "the Worker's spec doesn't allow jobs to do x."

So the Worker spec that you were thinking may not be needed here _is_ needed, because it allows the code under test to clear all of those hurdles and reach the point in the code where redaction actually happens.

If we wanted to get by with a more minimal Event for this test case (or others), we'd have to also define a Job for each _individual_ test case instead of always using one that has all the bells and whistles. That would work, but my gut says in the end it's a similar amount of code, but more complexity.

Reply:
Ah, ok, thanks for the explanation.  Current approach sgtm!